### PR TITLE
feature: allow cli to work on non-tty envs

### PR DESCRIPTION
**Pull request by geclos**
## Describe your changes
This allows the cli to run commands where user input is not expected. Of course it won't apply to use cases where user input is expected.

https://github.com/latitude-dev/latitude/assets/1948929/08d9841d-4b05-4ffc-869c-8674a068f330

## Checklist before requesting a review
- [x] I have added thorough tests
- [x] I have updated the documentation if necessary
- [x] I have added a human-readable description of the changes for the release notes
- [x] I have included a recorded video capture of the feature manually tested

### CODE REVIEW

File: .changeset/light-roses-begin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@latitude-data/cli": patch
+---
+
+Allows cli to run on non-tty environments
```
Comment (geclos) [inline .changeset/light-roses-begin.md:5]: remember that from now-on all main branch commits have to be a patch, regardless of semver semantics. Minor and major versions are reserved from next branches.

File: packages/cli/src/lib/telemetry/index.ts [hunk 2]
```diff
@@ -37,83 +39,48 @@ export class Telemetry {
...
 
-  enable() {
-    this.trackWithoutCheck({ event: 'telemetryEnabled' })
-    configStore.set('telemetry.enabled', true)
-  }
+  private async canTrack() {
+    if (this.enabled === undefined) return await this.askPermission()
 
-  disable() {
-    this.trackWithoutCheck({ event: 'telemetryDisabled' })
```
Comment (geclos) [inline packages/cli/src/lib/telemetry/index.ts:60]: hmm I've removed this events but i think we want them no? To compare non tracked users vs trakced users

**Review by andresgutgon — DISMISSED** [review]
Nice

**Review by csansoon — DISMISSED** [review]

File: packages/cli/src/lib/telemetry/index.ts (hunk 2 above)
Comment (andresgutgon) [reply packages/cli/src/lib/telemetry/index.ts:60]: I think it would be nice

**Pull request merged**